src/backend/catalog/external/externalmd.c

```diff
@@ -45,22 +46,25 @@
 #include "utils/numeric.h"
 #include "utils/guc.h"
 
-static HCatalogTable *ParseHCatalogTable(struct json_object *hcatalogMD);
-static void LoadHCatalogEntry(HCatalogTable *hcatalogTable);
-static Oid LoadHCatalogNamespace(const char *namespaceName);
-static void LoadHCatalogTable(Oid namespaceOid, HCatalogTable *hcatalogTable);
-static void LoadHCatalogType(Oid relid, Oid reltypeoid, NameData relname, Oid relnamespaceoid);
-static void LoadHCatalogDistributionPolicy(Oid relid, HCatalogTable *hcatalogTable);
-static void LoadHCatalogExtTable(Oid relid, HCatalogTable *hcatalogTable);
-static void LoadHCatalogColumns(Oid relid, List *columns);
+
+List *ParsePxfEntries(StringInfo json, char *profile, Oid dboid);
```

Comment:
are all of these functions static? if so, please add `static`, to make the code more clear.